### DISCUSSION TITLE
Users getting stuck while sending icon request!

### DISCUSSION THREAD

**Discussion by Harman-Thind** [zixpo/candybar#88] [Q&A] [answered]
So I have build an simple apk to send to my friends for icon request but while sending the request after selecting all the icons they are getting stuck on 'Building Request...' dialog with infinite loading. whereas I am able to send requests from my devices without any issue. What could be the problem here?

here's the screenshot of what they are getting - https://i.pinimg.com/originals/ab/6d/76/ab6d76db882a6f877b61319faac9d234.jpg

**Comment by sarsamurmu** [comment, marked as answer]
I guess the problem was with the new kind of adaptive icon (monet)

**Reply by sarsamurmu** [reply]
It's fixed now.
